### DISCUSSION TITLE
请问本地搜索功能该如何配置？

### DISCUSSION THREAD

**Discussion by kevinkelin** [hugo-next/hugo-theme-next#20] [Q&A]
我按照config.yaml 的配置将localSearch 打开， 可以点击这里的搜索还是不能弹出本地搜索框呢？ 
<img width="255" alt="image" src="https://user-images.githubusercontent.com/2595465/180245445-677adf76-6014-4301-9baa-eda9a1b02606.png">

我看作者的个人博客页 https://lisenhui.cn/  上是可以点击弹个搜索框的

请问这个该怎么配置呢？

**Comment by elkan1788** [comment]
那个是以前的主题，现在新主题的搜索功能暂时还没有实现，可关注  [文章搜索功能，本地搜索和Algolia在线搜索 #18](https://github.com/hugo-next/hugo-theme-next/issues/18) 更新动态。
